roles/bootstrap-os/tasks/bootstrap-centos.yml

```diff
@@ -4,6 +4,17 @@
     gather_subset: '!all'
     filter: ansible_distribution_*version
 
+- name: Add proxy to yum.conf or dnf.conf if http_proxy is defined
+  ini_file:
+    path: "{{ ( (ansible_distribution_major_version | int) < 8) | ternary('/etc/yum.conf','/etc/dnf/dnf.conf') }}"
```

Comment:
note for me: dnf is newer package management system than yum and it is available on centos8.